README.md

```diff
@@ -709,11 +743,13 @@ using Snabbdom.
 
 ## Common errors
 
-```
+```text
 Uncaught NotFoundError: Failed to execute 'insertBefore' on 'Node':
     The node before which the new node is to be inserted is not a child of this node.
 ```
```

Comment:
Something remark does that I found peculiar is that is a code block does not have a language specified, it uses indentation instead of fences. So I added a language here.